packages/shared/src/utils/os/os_detect.ts

```diff
@@ -89,3 +105,33 @@ export const mobileOSDetect = () => {
 
     return 'unknown';
 };
+
+export const mobileOSDetectAsync = async () => {
+    const userAgent = navigator.userAgent || navigator.vendor || window.opera || '';
```

Comment:
Vendor is deprecated. we need to remove it

Reply:
done...